tests/unit/test_protocol.py

```diff
@@ -31,27 +31,45 @@ def test_column_character_varying():
     assert protocol.build_character_varying("wuteuef") == b'\x00\x00\x00\x07wuteuef'
 
 
+def test_column_text():
+    assert protocol.build_text("lorem ipsum") == b'\x00\x00\x00\x0Blorem ipsum'
+
+
 def test_column_date():
     assert protocol.build_date(date(2019, 1, 1)) == b'\x00\x00\x00\x04\x00\x00\x1b\x1c'
 
 
+def test_column_json():
+    assert protocol.build_json('{"value": 1234}') == b'\x00\x00\x00\x0F{"value": 1234}'
+
+
+def test_column_jsonb():
+    assert protocol.build_jsonb(b'{"value": -4321}') == b'\x00\x00\x00\x11\x01{"value": -4321}'
+
+
 def test_row_2():
```

Comment:
does this test has any sense? In previous test cases we have covered all protocol functions, here we're just testing bytes concatenation

Reply:
HA! `JSONB` is NOT same as string. Not additional \0x01 character.

Reply:
no, I meant `test_row_2` test